lib/rules/no-multi-comp.js

```diff
@@ -61,15 +60,15 @@ module.exports = {
         const list = components.list();
         let i = 0;
 
-        for (const component in list) {
-          if (!has(list, component) || isIgnored(list[component]) || ++i === 1) {
-            continue;
+        Object.keys(list).forEach(component => {
```

Comment:
`(component, i)` and then we don't need to `++` it, nor have `let i = 0` above.

Reply:
Can't do this because we only want to `++i` if `isIgnored` is false and doesn't short-circuit. I.e. we want to start reporting from the _second_ non-ignored component.

Reply:
Actually found a way to achieve this, also using `filter`.